concent_api/concent_api/settings/base.py

```diff
@@ -1,5 +1,12 @@
 import os
 
+from git import (
+    InvalidGitRepositoryError,
+    Repo,
+)
```

Comment:
You forgot to add this library to `requirements.txt` and to regenerate `requirements.lock`.

Reply:
This will be not removed if I use golem_messages.__version__